.vscode/launch.json

```diff
@@ -27,10 +27,10 @@
             "runtimeExecutable": "${execPath}",
             "args": [
                 "--extensionDevelopmentPath=${workspaceFolder}",
-                "--extensionTestsPath=${workspaceFolder}/out/tests/suite/index"
+                "--extensionTestsPath=${workspaceFolder}/out/src/tests/suite/index"
             ],
             "outFiles": [
-                "${workspaceFolder}/out/tests/**/*.js"
+                "${workspaceFolder}/out/src/tests/**/*.js"
```

Comment:
💡Please Note: I think we chatted about this, for wider readers - this change don't really make any functional problem with the code.